stable/kommander-karma/templates/cleanup-configmap.yaml

```diff
@@ -0,0 +1,27 @@
+{{- if .Values.karma.configMap.enabled }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: karma-configmap-cleanup
+  labels:
+{{ include "kommander-karma.labels" . | indent 4 }}
+  annotations:
+    "helm.sh/hook": pre-delete
+    "helm.sh/hook-weight": "-4"
+    "helm.sh/hook-delete-policy": hook-succeeded
+spec:
+  template:
+    metadata:
+      name: karma-configmap-cleanup
+    spec:
+      serviceAccountName: {{ .Values.kommanderServiceAccount }}
+      containers:
+        - name: kubectl
+          image: docker.io/bitnami/kubectl:1.14.1
```

Comment:
we should update this docker image we are using 1.16 or 1.15.5

Reply:
should I make that update everywhere this is used?